packages/recordio/src/read.js

```diff
@@ -0,0 +1,25 @@
+export default function read(input) {
```

Comment:
Let's invest a minute or two to adjust the implementation to further improve readability and performance.

Reply:
Invested